resources/options.js

```diff
@@ -153,11 +156,27 @@ function switchOptions() {
     }
 }
 
+// Saves the default options by saving an object.
+function saveDefaults() {
+    for (var i = 0; i < options.length; i++) {
+        defaultOptions[options[i]] = optionsElement(i).prop('checked');
+    }
+}
+
+// Sets the options to the default options by updating the chrome storage and the options page.
+function restoreDefaults() {
+    chrome.storage.sync.set(defaultOptions, function() {});
```

Comment:
I think it would be appropriate to use a Toast to notify the user that the defaults have been restored succesfully.